turborepo-tests/integration/fixtures/boundaries/apps/my-app/index.ts

```diff
@@ -5,3 +5,20 @@ import { Ship } from "ship";
 import { Ship } from "@types/ship";
 // Import package that is not specified
 import { walkThePlank } from "module-package";
+
+// Import from a package that is not specified, but we have `@boundaries-ignore` on it
+// @boundaries-ignore
```

Comment:
Should this have produced a change in a test fixture where a warning is printed?

Reply:
We don't snapshot warnings since they go to stderr. I'm not sure if we should, since that could be pretty brittle (and also makes debugging tests slightly annoying)